tests/test_playbooks/vars/server.yml.example

```
@@ -4,6 +4,7 @@ foreman_username: admin
 foreman_password: "changeme"
 foreman_server_url: "https://foreman.example.com"
 foreman_validate_certs: false
+foreman_organization: "Default Organization"
```

```suggestion
foreman_organization: "Default Organization"
```

@evgeni my understanding is that `make test-setup` will copy this to `tests/test_playbooks/vars/server.yml`.

but, do you prefer that I define this variable in my test playbook instead?

---

We've been using "Test Organization" in other tests and hard coding there.

---

given you don't use that anymore, mind yanking that change?